PersonStageThree/src/com/beyond/person/helper/SetterClass.java

```diff
@@ -0,0 +1,48 @@
+package com.beyond.person.helper;
+
+import com.beyond.person.core.exception.*;
+import com.beyond.person.core.model.BasePerson;
+
+public class SetterClass {
+    /**
+     * method for setting the values
+     *
+     * @param name
+     * @param lastName
+     * @param designation
+     * @param email
+     * @param gender
+     * @param nickname
+     * @param age
+     */
+    public void setPerson(String name, String lastName, String designation, String email,
+                          String gender, String nickname, String age) throws InvalidFieldException {
+
+        BasePerson personCreated = new BasePerson(name, lastName);
+
+        try {
```

Comment:
@maria-salmasian not good way to handle error, just validate all fields and throw exception

Reply:
changed it to only 1 try and catch